### PR TITLE
force the HTTPManager to listen on IPv4

### PR DESCRIPTION
**Pull request by birarda**
Fix for HTTPManager to listen only on `QHostAddress:IPv4`

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3e4a05a57bf7284537065288605f8ad597f6b226 available at: https://gist.github.com/64f1506a49b670137af8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5632.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5632.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0fbbfb77d1d66df91e5130613e5fec1b6606129b available at: https://gist.github.com/64f1506a49b670137af8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5632.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5632.dmg

File: libraries/embedded-webserver/src/HTTPManager.cpp [hunk 1]
```diff
@@ -171,9 +172,19 @@ void HTTPManager::isTcpServerListening() {
 
 bool HTTPManager::bindSocket() {
     qCDebug(embeddedwebserver) << "Attempting to bind TCP socket on port " << QString::number(_port);
-    if (!listen(QHostAddress::Any, _port)) {
+    
+    if (false) {
```
Comment (murillodigital) [inline libraries/embedded-webserver/src/HTTPManager.cpp]: debug stuff

Comment (birarda) [inline libraries/embedded-webserver/src/HTTPManager.cpp]: yikes, thanks

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4b95d54955f0264b826cf6a3ca4cc9c7dc29ad84 available at: https://gist.github.com/64f1506a49b670137af8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5632.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5632.dmg

**Pull request merged**